otlp/scripts/generate-modules.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+if ! command -v protoc >/dev/null 2>&1; then
+  printf '%s\n' 'protoc is not available' 1>&2
+  exit 1
+fi
+
+if ! command -v proto-lens-protoc >/dev/null 2>&1; then
```

Comment:
For us nix users, can you add `haskellPackages.proto-lens-protoc` here: https://github.com/iand675/hs-opentelemetry/blob/main/flake.nix#L98

We've already got `protoc`, so that's fine.

Reply:
done